Implementation and documentation of AdvertServe <amp-ad> tag

renegademike opened the pull request:
Here is my implementation and documentation of the `amp-ad` tag for @AdvertServe as indicated in #5364 

Fixes #5364.

jridgewell commented:
/to @lannka

lannka commented on examples/ads.amp.html:
```diff
@@ -277,6 +278,15 @@ <h2 id="aduptech">Ad Up Technology</h2>
+          data-client="tester"
+          data-pid=0
+          data-zid=68>
+    <div placeholder></div>
```
please add the `<fallback>` tag as well.

lannka commented on examples/ads-legacy.amp.html:
```diff
@@ -125,6 +125,14 @@
       data-adtest="1">
   </amp-ad>
 
+  <h2>AdvertServe</h2>
```
no change needed in this file

renegademike commented:
Ok, I've removed my changes to examples/ads-legacy.amp.html and added the <fallback> tag as requested.

I have also implemented the renderStart, noContentAvailable and reportRenderedEntityIdentifier APIs.

renegademike commented:
Thanks, I've resolved the merge conflicts.
